packages/esm-service-queues-app/src/patient-queue-metrics/clinic-metrics.component.tsx

```diff
@@ -27,11 +28,17 @@ function ClinicMetrics() {
   const [initialSelectedItem, setInitialSelectItem] = useState(() => {
     return !currentService?.serviceDisplay || !currentService?.serviceUuid;
   });
-  const { totalCount } = useQueueEntries({
-    service: currentService?.serviceUuid,
-    location: currentQueueLocation,
-    isEnded: false,
-  });
+
+  const { data } = useSWR<QueueEntryResponse>(
```

Comment:
You are no longer using `useQueueEntries` at all. I can't imagine that this works in cases where the number of entries exceeds the page size. I did my development and testing of this feature by creating 11 service queue entries and setting the page size to 3. I recommend you do something similar. There are a lot of edge cases that we need to make sure will continue to work.